QModManager/Patching/Patcher.cs

```diff
@@ -101,22 +111,57 @@ internal static void Patch()
                 var initializer = new Initializer(CurrentlyRunningGame);
                 initializer.InitializeMods(modsToLoad);
 
-                int loadedMods = 0;
-                int erroredMods = 0;
-                foreach (QMod mod in modsToLoad)
+                QMod[] loadedMods = modsToLoad.Where(m => m.IsLoaded).ToArray();
+                QMod[] skippedMods = modsToLoad.Where(m => !m.IsLoaded && m.Status < 0).ToArray();
+                QMod[] erroredMods = modsToLoad.Where(m => !m.IsLoaded && m.Status > 0).ToArray();
```

Comment:
Consider using [List<T>FindAll](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.findall?view=netframework-4.8) instead of Linq.

The `modsToLoad` are already in a List, so this method should be available.